samplelistener/listener/dispatcher_service.bal

```diff
@@ -2,46 +2,45 @@ import ballerina/http;
 import ballerina/jballerina.java;
 
 service class DispatcherService {
-   private SlackAppMentionHandlingService|SlackAppCreatedHandlingService serviceRef;
-   //private string verificationToken;
+     private map<GenericService> services;
 
-   isolated function init(SlackAppMentionHandlingService|SlackAppCreatedHandlingService serviceRef) returns error? { //Receive verification token
-        self.serviceRef = serviceRef;
-        //self.verificationToken = "9asdbas9009123nas1e2";
+   isolated function init() returns error? {
+        self.services = {};
```

Comment:
Initialization can be done in L5 itself. And the `init` method can be removed.

Reply:
Resolved